From a0c2916cb6d3a8681f37eb496758afebea656bf2 Mon Sep 17 00:00:00 2001
From: Ankitanvesh <ankit192812@rediffmail.com>
Date: 25 Jan, 2018 12:37:21 PM

Codechef code for Buggy Calculator problem BUGCAL

diff --git a/src/BUGCAL.java b/src/BUGCAL.java
new file mode 100644
index 0000000..5b828c5
--- /dev/null
+++ b/src/BUGCAL.java
@@ -0,0 +1,25 @@
+import java.util.*;
+ class BUGCAL {
+ public static void main(String []args){
+  try {
+     Scanner in=new Scanner(System.in);
+  int T=in.nextInt();
+  for(int j=0;j<T;j++){
+  int n=in.nextInt();
+  in.nextLine();
+  String str=in.nextLine();
+  int[] a=new int[n];
+  for(int i=0;i<n;i++)
+  {a[i]=in.nextInt();}
+  if(n==1 && str=="Dee")
+  {   if((a[0]%2)==0)
+      System.out.println("Dee");
+  else System.out.println("Dum");
+  }
+  else
+      System.out.println("Dum");
+  }
+ }catch(Exception e){return;}
+ }
+ }
+
